src/lib/components/Navbar.svelte

```diff
@@ -111,6 +112,7 @@ The navbar component.
                              */
                             props: { options: books.map((b) => b.bookCode) /*bookAbbreviations*/ }
                         },
+                        // 
                         Chapter: {
```

Comment:
Chapter and Verse tabs should be removed here. They do not match the implementation in the Android Application.